packages/blade/src/components/BottomSheet/__tests__/__snapshots__/BottomSheet.web.test.tsx.snap

```diff
@@ -142,7 +142,7 @@ exports[`<BottomSheet /> should compose with DropdownButton 1`] = `
   -ms-flex-direction: column;
   flex-direction: column;
   padding-right: 8px;
-  padding-left: 8px;
+  padding-left: 0px;
```

Comment:
Is this expected? Wouldn't this visually look broken on BottomSheet, Dropdown, etc?

Reply:
"Yes... I think during the list view, there were many changes, so I missed this part.

Reply:
ohhh got it. So it was 0px before then became 8px during list view and that we're fixing now. Understood.